tests/testthat/test-target_market_share.R

```diff
@@ -488,7 +488,7 @@ test_that("w/ known input, outputs `technology_share` as expected (#184, #262)",
     c(0.6875, 0.3125)
   )
 
-  FIXME <- 1e-2  # Do we need 1e-3?
+  FIXME <- 1e-2 # Do we need 1e-3?
```

Comment:
nit: would rather see a useful function name and use FIXME in a comment

Reply:
Will address this also in a separate PR